builtin/fstk/buttonbar.lua

```diff
@@ -50,21 +50,17 @@ local function buttonbar_formspec(self)
 			(self.orientation == "horizontal" and
 			(btn_pos.x + self.btn_size <= self.pos.x + self.size.x)) then
 			
-		local borders="true"
+		local borders = "true"
 		
-		if self.buttons[i].image ~= nil then
-			borders="false"
+		if self.buttons[i].image ~= "" then
+			borders = "false"
 		end
-			
 		formspec = formspec ..
 			string.format("image_button[%f,%f;%f,%f;%s;%s;%s;true;%s]tooltip[%s;%s]",
 					btn_pos.x, btn_pos.y, self.btn_size, self.btn_size,
-					self.buttons[i].image, btn_name, self.buttons[i].caption,
+					self.buttons[i].image:gsub("\\\\", "/"), btn_name, self.buttons[i].caption,
```

Comment:
That has already been done earlier.
https://github.com/minetest/minetest/blob/master/builtin/mainmenu/tab_singleplayer.lua#L60
An example file path of `self.buttons[i].image`:
\\Minetest\\games\\my_game\\menu\\icon.png

I tried adding the drive letter in front of that but it did not solve the problem.